.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,42 @@
+name: Docker
+
+on:
+  push:
+    # Publish `main` as Docker `latest` image.
+    branches:
+      - main
+
+env:
+  IMAGE_NAME: al-showit
```

Comment:
Nevim, zda ma cenu to smesovat s vyrobni linkou - chapu to jako apku mimo VL. Je to dane tim GH a tim, ze nemame dve nasazeni. Muzeme treba prefixovat sluzby nad publikovanou verzi slovniku jako 'ssp-', nebo jinak.

Co myslite?

Reply:
Jasny, dal jsem ten prefix pryc.

Reply:
A nebude to ještě vadit v tom druhém PRku? .env.docker-compse obsahuje ID=al-showit

Reply:
Ne, je to jedno :-) Ten docker-compose slouží jenom pro lokální testování a nijak nesouvisí s imagem, který se publikuje na GitHub.